test/lando-config-string-replacement.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Tests for lando config file string replacement.
+ * @file lando-config-string-replacement.js
+ */
+
+'use strict';
+
+var assert = require('assert');
```

Comment:
@dustinleblanc i think you should decide and then make sure we all obey what you decide. i've been using `should` and `expect` thus far so i think i'm in compliance. worth a doc update?

Reply:
I think should/expect are both fine, but if we stick to the bdd api, that will keep tests easier to reason about since they will all be in the same style. I agree, documentation update is warranted. I just opened #766 to document this

Reply:
Done.

Reply:
Thanks!